src/workerd/api/node/tests/BUILD.bazel

```diff
@@ -240,6 +243,24 @@ sh_test(
     ],
 )
 
+sh_test(
+    name = "process-stdio-fs-sh-test",
+    srcs = ["test_process_stdio.sh"],
+    args = [
+        "$(location //src/workerd/server:workerd_cross)",
+        "$(location process-stdio-fs-nodejs-test.wd-test)",
+        "$(location process-stdio-fs-nodejs-test.expected_stdout)",
+        "$(location process-stdio-fs-nodejs-test.expected_stderr)",
+    ],
+    data = [
+        "process-stdio-fs-nodejs-test.expected_stderr",
+        "process-stdio-fs-nodejs-test.expected_stdout",
+        "process-stdio-fs-nodejs-test.js",
+        "process-stdio-fs-nodejs-test.wd-test",
+        "//src/workerd/server:workerd_cross",
```

Comment:
What's workerd_cross? I can't find many references to it in our codebase.

Reply:
AFICT, it's the base build file used by all wd_test tests, which we need to explicitly include for these "sh" style tests.